### PR TITLE
finished with coding challenge

### PR DESCRIPTION
**Pull request by devonhackley**

### CODE REVIEW

**Comment by devonhackley** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java [hunk 1]
```diff
@@ -162,5 +162,57 @@ public void testLinkedListNumLessThanList(){
...
+
+
+        libraryForTest2.append(5);
+        libraryForTest2.append(9);
+        libraryForTest2.append(4);
+
+        LinkedList.mergeLists(libraryForTest, libraryForTest2);
+
+        Node node = new Node(5);
+        assertEquals(node.data, libraryForTest.head.next.data);
```
Comment (evanslaton) [inline code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java]: This isn't checking to make sure the entire LinkedList is correct!
